packages/frontend-shared/src/locales/en-US.json

```diff
@@ -739,10 +739,12 @@
         "link": "guide on recording runs"
       },
       "gitRepositoryNotDetected": "Git repository not detected",
-      "ensureGitSetupCorrectly": "Cypress uses Git to associate runs with your local state. Please ensure that version control is set up correctly.",
+      "ensureGitSetupCorrectly": "Cypress uses git to associate runs with your local state. Please ensure that git is properly configured for your project.",
       "learnAboutRecordingSrText": "about recording a run to Cypress Cloud",
       "learnAboutDebuggingSrText": "about debugging CI failures in Cypress",
       "learnAboutProjectSetupSrText": "about project setup in Cypress",
+      "noRunsFoundForBranch": "No runs found for your branch",
+      "noRunsForBranchMessage": "Cypress uses Git to show runs for your branch. Ensure that version control is properly configured and that you are sending Git information to Cypress Cloud.",
```

Comment:
Why uppercase `Git` here, but lowercase in message above?

Reply:
This is the way it was in the ticket. @warrensplayer should the requirements be changed?

Reply:
"Git" should be capitalized.  I will get the requirements and Figma updated.

Reply:
Screenshot in issue has been updated